extensions/bond/sawtooth_bond/txn_family.py

```diff
@@ -49,6 +50,19 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def _load_config(journal):
+    try:
+        with open('./etc/sawtooth-bond.json') \
+                as bond_fd:
+            bond_config = json.load(bond_fd)
+    except IOError, ex:
+        LOGGER.exception('IOError: %s', str(ex))
+        return
+
+    libor_publisher_addr = bond_config["LiborPublisherAddr"]
+    CreateLIBORUpdate.LIBOR_PUBLISHER_ADDR(libor_publisher_addr)
```

Comment:
Am I missing something?  LIBOR_PUBLISHER_ADDR has been been made a global variable in sawtooth_bond.updates.libor and here you are accessing it like a function on the CreateLIBORUpdate class.